subworkflows/local/longread_coverage.nf

```diff
@@ -138,28 +138,31 @@ workflow LONGREAD_COVERAGE {
     //         EMITS A MERGED BAM
     SAMTOOLS_MERGE(
         collected_files_for_merge,
-        reference_tuple.map { it[1] }, 
-        MINIMAP2_INDEX.out.index.map { it[1] }
+        reference_tuple, 
+        MINIMAP2_INDEX.out.index
```

Comment:
For samtools merge, the input options are input files, fasta and fai. How is `minimap_index` equivalent to `fai`? And why do you need `minimap_index`? I thought it was more efficient to do it on the fly.

Reply:
They are not. 
This is an error by the writer and has now been replaced with `[[].[]]`

It won't need an index file as we aren't using cram here.